Pod/Classes/MenuView.swift

```diff
@@ -8,6 +8,8 @@
 
 import UIKit
 
+internal let MenuViewItemsMarginRigth: CGFloat = 20
```

Comment:
Thanks for you answer. Then this static margin wouldn't be appropriate way, because this change is supposed to be for dynamic change of the number of items. In addition, I'm not sure this approach would be suitable for other cases as well. 

Ideally there should be method which can change it's option, but it's still in progress...
Well, let me think different approach.